<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScenePainter</class>
 <widget class="QWidget" name="GameScenePainter">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tetris</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/tetris-icon-25.jpg</normaloff>:/image/tetris-icon-25.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#GameScenePainter {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                 stop:0 #B2B9BE,
                                 stop:1 #2F4052);
    border-radius: 10px;             /*四角圆角的直径*/
    border-style:solid;             /*边框样式（实线）*/
    border-width:2px;             /*边框宽度*/
    border-color:rgb(213, 213, 213);     /*边框颜色*/
}</string>
  </property>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>32</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: transparent; /*背景色*/ 
	border-style: outset;
	border-width: 2px;
	border-radius:5px; /*边界圆滑*/
	border: none;
	min-width:2em;
	padding: 5px;
	image: url(:/image/close.png);
}
 
QPushButton:hover
{
	background-color: rgba(255,0,0,50);
}
 
QPushButton:pressed
{
	background-color: rgba(165,42,42,75); 
	border-style: inset;
}
QPushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ScoreLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素18</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>得  分</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ScoreLine">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>95</y>
     <width>120</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LCDMono2</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background: black;
color:white;
border: none;
 border-radius: 10px;             /*四角圆角的直径*/
border-style:solid;             /*边框样式（实线）*/
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="NextBlockLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素18</family>
     <pointsize>15</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>下一个方块</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="SpeedLine">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LCDMono2</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background: black;
color:white;
border: none;
 border-radius: 10px;             /*四角圆角的直径*/
border-style:solid;             /*边框样式（实线）*/
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="SpeedLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正像素18</family>
     <pointsize>15</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>当前下落速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>570</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgba(0,153,255,75); /*背景色*/ 
	border: transparent;
	border-style: outset;
	border-width: 1px;
	border-radius: 10px; /*边界圆滑*/
	font: bold 18px;
	min-width:2em;
	color: black; /*字体颜色*/
	font-family: 方正像素18;
	padding: 2px;

}
 
QPushButton:hover
{
	background-color: rgba(0,141,235,75);
}
 
QPushButton:pressed
{
	background-color: rgba(0,129,214,80); 
	border-style: inset;
}
QPushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>保存游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PARButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgba(0,153,255,75); /*背景色*/ 
	border: transparent;
	border-style: outset;
	border-width: 1px;
	border-radius: 10px; /*边界圆滑*/
	font: bold 18px;
	min-width:2em;
	color: black; /*字体颜色*/
	font-family: 方正像素18;
	padding: 2px;

}
 
QPushButton:hover
{
	background-color: rgba(0,141,235,75);
}
 
QPushButton:pressed
{
	background-color: rgba(0,129,214,80); 
	border-style: inset;
}
QPushButton:!enabled{
	background-color: rgb(100, 100, 100);
	border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>暂停游戏</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
